plugins/woo-ai/src/constants.ts

```diff
@@ -1,3 +1,2 @@
-export const MIN_TITLE_LENGTH = 20;
 export const MAX_TITLE_LENGTH = 200;
-export const MIN_TITLE_LENGTH_FOR_DESCRIPTION = 20;
+export const MIN_TITLE_LENGTH_FOR_DESCRIPTION = 15;
```

Comment:
The title generation still comes in below 20 characters at times (not sure GPT 3.5 can count) so I'm lowering this.